agent-connection/src/test/kotlin/com/luxoft/blockchainlab/corda/hyperledger/indy/PythonRefAgentConnectionTest.kt

```diff
@@ -25,7 +28,8 @@ class PythonRefAgentConnectionTest {
 
     class InvitedPartyProcess (
             private val agentUrl: String,
-            val proofSchemaId: String = "${Random().nextInt()}:::1"
+            val proofSchemaId: String = "${Random().nextInt()}:::1",
+            val tailsHash: String = "${Random().nextInt(Int.MAX_VALUE)}"
```

Comment:
JFYI.
To emulate real hash value using random you can use this: 
```kotlin
val hash = ByteArray(256)
Random.nextBytes(hash)
```

Reply:
nevermind

Reply:
ok, disregarded